bottom-bar/src/main/java/com/roughike/bottombar/BottomBar.java

```diff
@@ -141,6 +143,11 @@ private void populateAttributes(Context context, AttributeSet attrs) {
             tabXmlResource = ta.getResourceId(R.styleable.BottomBar_bb_tabXmlResource, 0);
             isTabletMode = ta.getBoolean(R.styleable.BottomBar_bb_tabletMode, false);
             behaviors = ta.getInteger(R.styleable.BottomBar_bb_behavior, BEHAVIOR_NONE);
+
+            if (isIconsOnlyMode() && isShiftingMode()) {
```

Comment:
why is this restriction? they might want to set it as icon only and shifting and set the text manually as you did on your example, or is not that how it works your example?